ledger/participant-state/src/main/scala/com/daml/ledger/participant/state/v2/CompletionInfo.scala

```diff
@@ -31,15 +31,19 @@ import com.daml.logging.entries.{LoggingValue, ToLoggingValue}
   *                               this deduplication information, it MUST adhere to the deduplication
   *                               guarantee under a sensible interpretation of the corresponding
   *                               [[CompletionInfo.optDeduplicationPeriod]].
+  *
   * @param submissionId           An identifier for the submission that allows an application to
   *                               correlate completions to its submissions.
+  *
+  *                                Optional as entries created by the participant.state.v1 API do not have this filled.
```

Comment:
Yes, I actually liked more like this because it's visually indicative that this text still is part of the submissionId documentation but I agree that being consistent is more important so I'll remove the extra indentation.